addons/stock_sms/models/stock_picking.py

```diff
@@ -46,9 +46,12 @@ def _action_generate_warn_sms_wizard(self):
         }
 
     def _sms_get_number_fields(self):
-        """ This method returns the fields to use to find the number to use to
-        send an SMS on a record. """
-        return ['mobile', 'phone']
+        """ No phone or mobile field is available on ``stock.picking`` model. Instead SMS will
+        fallback on partner-based computation using ``_sms_get_partner_fields``. """
+        return []
+
+    def _sms_get_partner_fields(self):
+        return ['partner_id']
```

Comment:
This should be a different commit for `stock_sms` as it's a different fix.

Reply:
Hi @thiryr , a different commit in the same PR, or do I need to create a new one ?

Reply:
Just in the same PR, linking the same task :)

It's just so we can clearly separate and justify each change

Reply:
Done, hope the commit message is clear enough :)

Reply:
Makes sense to me :D

Although note I think the guidelines are normally 50 characters for the title and 72 for the commit lines, including `[TAG] module_name:`

Reply:
Tried to reduce it the most :)